### PR TITLE
mocha-framework: add file property

### PR DESCRIPTION
**Pull request by ablok**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
Adds the file property to the mocha framework suite object. It seems to be available and it was always available in V4. So I don't know if it's omission was on purpose or just an oversight

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-mocha-framework/src/index.js [hunk 1]
```diff
@@ -166,16 +166,17 @@ class MochaAdapter {
         if (params.payload) {
             message.title = params.payload.title
             message.parent = params.payload.parent ? params.payload.parent.title : null
-
+
```
Comment (christian-bromann) [inline packages/wdio-mocha-framework/src/index.js]:
```suggestion

```

File: packages/wdio-mocha-framework/src/index.js [hunk 2]
```diff
@@ -166,16 +166,17 @@ class MochaAdapter {
...
-
+            
             /**
              * get title for hooks in root suite
              */
             if (message.parent === '' && params.payload.parent && params.payload.parent.suites) {
                 message.parent = params.payload.parent.suites[0].title
             }
-
+
```
Comment (christian-bromann) [inline packages/wdio-mocha-framework/src/index.js]:
```suggestion

```

Comment (ablok) [reply packages/wdio-mocha-framework/src/index.js]: I added the file property to the unit tests.

Comment (christian-bromann) [reply packages/wdio-mocha-framework/src/index.js]: @ablok please also remove the spaces as suggested above

Comment (ablok) [reply packages/wdio-mocha-framework/src/index.js]: @christian-bromann Wow, I really thought it already did that after I committed your suggested change. Well, shoud be fixed now.

**Comment by ablok** [comment]
@christian-bromann all done

**Pull request merged**